src/app/file-manager/file-export-modal/file-export-modal.component.ts

```diff
@@ -91,17 +101,17 @@ export class FileExportModalComponent implements OnInit {
     }
   }
 
-  formChange(): boolean {
+  disableExportButton(): boolean {
```

Comment:
With this change do you still need `disableExport` as a field? It looks like you compute it completely in this method. If so, you can make `disableExport` a local var and remove the field and references to it in this class.